test/unit/spec/common/cellComponents/tabs/jobActionDropdown-Spec.js

```diff
@@ -3,19 +3,16 @@ define([
     'common/jobMessages',
     'common/jobs',
     'common/props',
-    '/test/data/testAppObj',
-], (JobActionDropdown, JobMessages, Jobs, Props, TestAppObj) => {
+    '/test/data/jobsData',
+], (JobActionDropdown, JobMessages, Jobs, Props, JobsData) => {
```

Comment:
I switched out using the TestAppObj for creating a model using allJobsWithBatchParent